blocks/address/src/app.tsx

```diff
@@ -499,8 +520,15 @@ export const App: BlockComponent<RootEntity> = ({
                   }
                 }}
               >
-                <Box sx={{ display: "flex", gap: 1.5 }}>
+                <Box sx={{ display: "flex", gap: 1.5, maxWidth: 420 }}>
```

Comment:
Nit: would be nice to capture the max width of the input in a named variable, and make use of it here

Reply:
Done in 59259c798be853ed9062b9068a9602ef82f39dab